src/integrationTest/resources/application.yml

```diff
@@ -84,7 +84,7 @@ security:
 
 idam:
   s2s-auth:
-    totp_secret: ${S2S_SECRET:}
+    totp_secret: ${JUDICIAL_API_S2S_SECRET:}
     microservice: rd_judicial_api
```

Comment:
I think you missed to remove this https://github.com/hmcts/rd-judicial-api/blob/9769f5fba0e1dd171188244e861d50bab4b300f1/src/integrationTest/resources/application.yml#L38

Reply:
This cant be removed since key vault secret for integration test cases does not work

Reply:
Fair enough!!